src/DOM/HTML/Indexed.purs

```diff
@@ -328,6 +329,7 @@ type HTMLinput = Interactive
   ( accept :: MediaType
   , autocomplete :: OnOff
   , autofocus :: Boolean
+  , capture :: Boolean
```

Comment:
Per https://w3c.github.io/html-media-capture/#the-capture-attribute, `capture` is a sensible enum. There should be an appropriate data type for this, like the other enumerated values.

```purescript
data CaptureValue
  = CaptureUser
  | CaptureEnvironment
```